SmartDeviceLink/private/SDLVoiceCommandUpdateOperation.m

```diff
@@ -20,6 +20,7 @@
 @interface SDLVoiceCommand()
 
 @property (assign, nonatomic) UInt32 commandId;
+@property (copy, nonatomic, readwrite) NSArray<NSString *> *voiceCommands;
```

Comment:
Instead of doing the validation of the empty strings in `SDLVoiceCommandUpdateOperation` and assigning it back, why don't we just do the empty string validation in the `SDLVoiceCommandManager`'s `setVoiceCommands:` method?